website/docs/cli/cloud/settings.mdx

```diff
@@ -1,32 +1,53 @@
 
-After adding or changing a `cloud` block, you must run `terraform init`.
+- [State overview](/terraform/language/state) in the Terraform configuration language reference.
+- [Terraform State in HCP Terraform](/terraform/cloud-docs/workspaces/state) in the HCP Terraform documentation.
 
-## The `cloud` Block
+Complete the following steps to connect to HCP Terraform:
 
-The `cloud` block is a nested block within the top-level `terraform` settings
-block. It specifies which HCP Terraform workspaces to use for the current
-working directory.
+1. Provide credentials to HCP Terraform.
+1. Define connection settings in your Terraform configuration.
+1. Initialize the working directory.
+1. Migrate state data. This step is optional.
+
+## Requirements
+
+You must have a user profile in HCP Terraform with permissions to create a workspace. Refer to [Workspace Permissions](/terraform/cloud-docs/users-teams-organizations/permissions) in the HCP Terraform documentation for additional information.
+
+## Provide credentials
+
+You must provide credentials to access HCP Terraform. We recommend using the
+[`terraform login`](/terraform/cli/commands/login) command to log into Terraform. You can also provide a user token in the Terraform configuration. Refer to the [`token`](/terraform/language/terraform#terraform-cloud-token) attribute in the Terraform configuration reference for additional information.
+
+
+## Define connection settings
+
+Add a `cloud` block to your Terraform configuration and configure the connection settings to link the working directory to an HCP Terraform workspace. The `cloud` block is a member of the `terraform` block. Refer to the [`terraform` block reference](/terraform/language/terraform) for additional information.
+
+Specify the following settings in the `cloud` block:
+
+- `organization`:  Specifies the name of an HCP Terraform organization to connect to.
+- `workspaces.tags`: Specifies a list of single-value string tags. Terraform links the working directory to existing workspaces in the organization that have matching tags. If there are no existing workspaces with matching tags, the Terraform CLI prompts you to create a new workspace that inherits the tags you specify in this field when you initialize the configuration.
+- `workspaces.name`: You can specify the name of an existing workspace to associate with the Terraform configuration instead of using tags. If you configure the `name`, you cannot use the `tags` configuration.
+- `workspaces.project`: You can specify the name of an existing project to limit associating the configuration with workspaces in the specific project.
```

Comment:
This could be a little clearer. The way it was explained to me is that specifying the project associates the configuration with the all workspaces in the project that have the tags. It's a filtering mechanism so that you don't have to figure out which specific workspaces to associate the configuration with. I don't know how realistic this example is, but if you have a project called 'test' and maybe a set of workspaces tagged with `api`, then you could direct Terraform to associate the configuration with just those workspaces. 

We'll need to get some clarity from someone on what happens if the project does not exist, but Terraform creates a workspace using either the `name` or the `tags` value if they do not exist in the specified project. 

Some of this is already explained in the reference. If you are following this instructions, you might not have some of these questions because you already know that there is a project, workspace, or set of workspaces in HCP that you want to connect to. To answer those questions, we expect someone to follow the link to the complete reference.

Reply:
```suggestion
- `workspaces.project`: You can specify the name of an existing project. Terraform associates the configuration with workspaces in the project that match the `name` or `tags`. 
```